.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     camelcase: 0,
     // semi                                : ['error', 'always'],
     'array-callback-return': 0,
+    'arrow-body-style': 0,
```

Comment:
Can we use `as-needed` instead for this rule?

Reply:
```suggestion
    'arrow-body-style': 'as-needed',
```